tests/ut-torchapi.cc

```diff
@@ -44,6 +44,11 @@ static std::string bert_train_repo
 static std::string bert_train_data
     = "../examples/torch/bert_training_torch_140_transformers_251/data/";
 
+static std::string sinus = "../examples/all/sinus/";
+
+static std::string iterations_nbeats_cpu = "100";
```

Comment:
FYI nbeats tests are not built on CPU because they are enclosed within the scope of a `#if !defined(CPU_ONLY)` that was originally put in place to remove BERT training tests from clogging CPU builds. We may want to add nbeats training to CPU unit tests, what do you think ?

Reply:
ok, done